src/main/java/edu/indiana/dlib/amppd/controller/FileUploadController.java

```diff
@@ -147,49 +136,47 @@ public PrimaryfileSupplement handlePrimaryfileSupplementUpload(@PathVariable("id
 //        return "redirect:/";
     }
 
-    @PostMapping("/datalib/import/{id}")
-	@ResponseBody
-	public String HandleGalaxyImport(@PathVariable("id") String lib_name){
-
-		final String galaxyInstanceUrl = config.getBaseUrl();
-
-		final String galaxyApiKey = galaxyApiService.getApiKey();
-		GalaxyInstance galaxyInstance = GalaxyInstanceFactory.get(galaxyInstanceUrl, galaxyApiKey, true);
-
-		final LibrariesClient libraryClient = galaxyInstance.getLibrariesClient();
-
-		String msg = "Galaxy Instance: "+galaxyInstanceUrl+"" +
-				"t Lib:"+libraryClient.getLibraries().toString();
-
-		final List<Library> libraries = libraryClient.getLibraries();
-		Library matchingLibrary = null;
-		for(final Library library : libraries) {
-			if (library.getName().equals(lib_name)) {
-				matchingLibrary = library;
-			}
-		}
-
-		if (!matchingLibrary.equals(null)) {
-
-			final LibraryContent rootFolder = libraryClient.getRootFolder(matchingLibrary.getId());
-			final FilesystemPathsLibraryUpload upload = new FilesystemPathsLibraryUpload();
-			upload.setContent("/Users/aahmad/Desktop/new_b1_visa");
-			upload.setLinkData(true);
-			upload.setFolderId(rootFolder.getId());
-
-			final ClientResponse uploadResponse = libraryClient.uploadFileFromUrl(matchingLibrary.getId(), upload);
-
-			msg+="\t Upload Completed.";
-
-		} else {
-			log.info("Unable to find lib"+lib_name);
-		}
-
-		log.info(msg);
-		return msg;
-	}
-
-
+//    @PostMapping("/datalib/import/{id}")
```

Comment:
@yingfeng-iu  can we remove commented code?

Reply:
yes, defintely. i kept them in case there will be changes reverted and such. will remove before merge